scripts/ci/check_compliance.py

```diff
@@ -971,6 +974,26 @@ def run(self):
                              f"less than {limit >> 10}kB")
 
 
+class MaintainersFormat(ComplianceTest):
+    """
+    Check that MAINTAINERS.yml parses correctly.
+    """
+    name = "MaintainersFormat"
+    doc = "Check that MAINTAINERS.yml parses correctly."
+    path_hint = "<git-top>"
+
+    def run(self):
+        MAINTAINERS_FILE = "MAINTAINERS.yml"
```

Comment:
You should allow `MAINTAINERS.yaml` as well. See here for an example:
https://github.com/zephyrproject-rtos/zephyr/blob/e7a4aec3c9f3bd45dee5993cbbf88597341d5d30/scripts/zephyr_module.py#L159-L160

Reply:
Cool, reworked to accept both.